engine/crates/parser-postgresql/src/registry/queries/input/oneof.rs

```diff
@@ -30,6 +30,7 @@ pub(crate) fn register(input_ctx: &InputContext<'_>, table: TableWalker<'_>, out
 
                 let query_name = type_prefix.to_camel_case();
 
+                builder.map_unique_constraint(&query_name, constraint.id());
```

Comment:
Won't this break with two tables `A` and `B` having both a constraint on columns with the same name?

Reply:
This builder is per table, meaning there will be no other tables managing this collection. In the `Names` struct, the mapping is per `(TableId, StringId)`, so it's not a problem.